build/dependencies.props

```diff
@@ -1,6 +1,13 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
```

Comment:
Unfortunately, we have to keep in the repo. Otherwise, opening a clean repo would break in VS.